src/Http/Controllers/API/ApiController.php

```diff
@@ -23,8 +24,16 @@ class ApiController extends Controller
      *
      * @param  mixed  $item
      */
-    protected function abortIfUnpublished($item)
+    protected function abortIfUnpublished(Request $request, $item)
     {
+        if ($request->statamicToken()) {
```

Comment:
Check that its a live preview token, not just any token. You can use `$request->isLivePreview()`.